titan/agent.py

```diff
@@ -3,7 +3,13 @@
 
 from typing import Dict, Set, Optional, Iterator, Iterable
 
-from .utils import safe_divide, safe_dist, get_independent_bin
+from .utils import (
+    safe_divide,
+    safe_dist,
+    get_independent_bin,
+    safe_random_choice,
+    safe_rand_int,
```

Comment:
I know the names for random choice and rand int are based on function names in packages, but can we choose either `random` or `rand` for both?